synapse/handlers/groups_local.py

```diff
@@ -284,27 +295,7 @@ async def create_group(self, group_id, user_id, content):
             local_attestation = None
             remote_attestation = None
         else:
-            local_attestation = self.attestations.create_attestation(group_id, user_id)
-            content["attestation"] = local_attestation
-
-            content["user_profile"] = await self.profile_handler.get_profile(user_id)
-
-            try:
-                res = await self.transport_client.create_group(
-                    get_domain_from_id(group_id), group_id, user_id, content
-                )
```

Comment:
[`TransportLayerClient`](https://github.com/matrix-org/synapse/blob/30fba6210834a4ecd91badf0c8f3eb278b72e746/synapse/federation/transport/client.py#L33) does not have a `create_group`, since the groups code is to be replaced it seemed prudent to just fail quickly here.